### PR TITLE
add pro/beta page

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- added /pro/beta, according to the [copy doc](https://docs.google.com/document/d/1GFVENQ9g5U4FSyqZHboXVEQ_RHKxRSwq3NC9PWrwURs/edit#) and the [design](https://www.figma.com/file/ZAihKxF2wmj9wvoPSnPKmY/Commercial%3A-UA-%2Fsubscribe?node-id=1571%3A14153)

## QA

- Visit https://ubuntu-com-12099.demos.haus/pro/beta
- See that it matches the design and the copy doc

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6046

### CODE REVIEW

File: templates/pro/beta.html [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+    </div>
+  </section>
+  <section class="p-strip is-deep u-no-padding--top">
+    <hr class="is-fixed-width u-no-margin--bottom">
+    <div class="row">
+      <div class="col-3 col-medium-1">
+        <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">Overview</h2>
+      </div>
+      <div class="col-9 col-medium-5">
+        <p>New to Ubuntu Pro? This how-to guide will help you to understand how to activate your Ubuntu Pro subscription and choose which services to enable.
```
Comment (minkyngkm) [inline templates/pro/beta.html:23]:
```suggestion
        <p>New to Ubuntu Pro? This how-to guide will help you understand how to activate your Ubuntu Pro subscription and choose which services to enable. Together, we will identify security updates available uniquely with an Ubuntu Pro subscription, and we will apply fixes. 
```

File: templates/pro/beta.html [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+    <hr class="is-fixed-width u-no-margin--bottom">
+    <div class="row">
+      <div class="col-3 col-medium-1">
+        <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">Overview</h2>
+      </div>
+      <div class="col-9 col-medium-5">
+        <p>New to Ubuntu Pro? This how-to guide will help you to understand how to activate your Ubuntu Pro subscription and choose which services to enable. 
+        </p>
+        <p>
+          We will start by getting an Ubuntu Pro personal subscription. We will then attach this subscription to your existing Ubuntu LTS machine and enable the Expanded Security Maintenance for Applications (ESM Apps) service to find out if any additional security fixes are available for you.
```
Comment (minkyngkm) [inline templates/pro/beta.html:26]:
```suggestion
       We will start by getting a free, personal subscription. Then we will attach this subscription to your existing Ubuntu LTS machine and enable the Expanded Security Maintenance for Applications (esm-apps) in beta, to find out if any additional security fixes are available for you on your machine.

```

File: templates/pro/beta.html [hunk 3]
```diff
@@ -0,0 +1,464 @@
...
+        </p>
+        <p>
+          Including Ansible, Apache Tomcat, Apache Zookeeper, Docker, Drupal, Nagios, Node.js, phpMyAdmin, Puppet, PowerDNS, Python 2, Redis, Rust, WordPress, and many more...
+        </p>
```
Comment (minkyngkm) [inline templates/pro/beta.html:30]:
```suggestion

```

File: templates/pro/beta.html [hunk 4]
```diff
@@ -0,0 +1,464 @@
...
+        <p>
+          <a href="https://ubuntu.com/pro">Want to learn more about the benefits of Ubuntu Pro?</a>
```
Comment (minkyngkm) [inline templates/pro/beta.html:32]:
```suggestion
          <a href="https://ubuntu.com/pro">Want to learn more about the benefits of Ubuntu Pro before moving on?</a>
```

File: templates/pro/beta.html [hunk 5]
```diff
@@ -0,0 +1,464 @@
...
+          <li class="p-list__item">
+            <span class="u-max-width">
+              Sudo access
+            </span>
+          </li>
+          <li class="p-list__item">
+            <span class="u-max-width">
+              An email adreess
+            </span>
+          </li>
```
Comment (minkyngkm) [inline templates/pro/beta.html:75]:
```suggestion
          </li>
          <li class="p-list__item">
            <span class="u-max-width">
              Ubuntu Pro client version 27.11.2 or newer
            </span>
          </li>
```

File: templates/pro/beta.html [hunk 6]
```diff
@@ -0,0 +1,464 @@
...
+          <li class="p-list__item">
+            <span class="u-max-width">
+              Sudo access
+            </span>
+          </li>
+          <li class="p-list__item">
+            <span class="u-max-width">
+              An email adreess
```
Comment (minkyngkm) [inline templates/pro/beta.html:73]:
```suggestion
              An email address, or an existing Ubuntu One account
```

**Comment by minkyngkm** [comment]
@sowasred2012 I had a quick look and there seem to be some updates on copy docs! :)

File: templates/pro/beta.html [hunk 7]
```diff
@@ -0,0 +1,586 @@
...
+    <div class="row">
+      <div class="col-3">
+        <p class="p-step-counter">
+          10
+        </p>
+      </div>
+      <div class="col-9">
+        <h2>What else can you use your Ubuntu Pro subscription for?</h2>
+
+        <p>For users running in regulated environments, we have a set of FIPS-certified crypto-modules and hardening scripts available. To enable them, consider enabling other Pro services you are entitled to, such as the Ubuntu Security Guide.</p>#
```
Comment (albertkol) [inline templates/pro/beta.html:555]:
```suggestion
        <p>For users running in regulated environments, we have a set of FIPS-certified crypto-modules and hardening scripts available. To enable them, consider enabling other Pro services you are entitled to, such as the Ubuntu Security Guide.</p>
```

**Review by albertkol — APPROVED** [review]
:+1: To me it looks good. Just removed a rogue hashtag.

**Pull request merged**